docs/core/common/assets/js/api/core/store.md

```diff
@@ -0,0 +1,128 @@
+## API - `$e.store`
+*  **Description**: `$e.store` API is a global state manager that allows to create global states for components.
+   It's actually a wrapper around [Redux Toolkit](https://redux-toolkit.js.org/introduction/getting-started), and it uses [Slices](https://redux-toolkit.js.org/api/createslice) under the hood.
+*  **Location**: *core/common/assets/js/api/core/store.js*
+*  **Methods**:
+
+| Method                        | Params                                                    | Returns                 | Description
+|-------------------------------|-----------------------------------------------------------|-------------------------|---------------------------------------------------------------------
+| `$e.store.register()`         | `{String}` *sliceId*, `{Slice}` *instance*   			    | `{void}`                | Register a new [Redux Toolkit Slice](https://redux-toolkit.js.org/api/createslice).
+| `$e.store.getAll()`           |                                                           | `{Object}`              | Get all existing Slices.
+| `$e.store.get()`              | `{String}` *sliceId* (optional)						    | `{Slice}` \| `{Object}` | Get a specific Slice instance by its ID, or get all of them if ID is empty.
+| `$e.store.getReduxStore()`    |                                                           | `{Object}`              | Get the actual [Redux Store](https://redux-toolkit.js.org/api/configureStore) object (mainly used for setting up a `<Provider />` in React).
+| `$e.store.getState()`         |                                                           | `{any}`                 | Get the [current global state tree](https://redux.js.org/api/store#getstate).
+| `$e.store.subscribe()`        | `{Function}` *listener*                                   | `{function}`            | [Subscribe](https://redux.js.org/api/store#subscribelistener) to state changes.
+| `$e.store.dispatch()`         | `{Object}` *action*                                       | `{Object}`              | [Dispatch](https://redux.js.org/api/store#dispatchaction) a Redux action.
+
+
+## Guidelines & Conventions:
+* Each Slice is owned by a [component](./components.md#guidelines-conventions--files-structure).
+* Each [component](./components.md#guidelines-conventions--files-structure) can override the `defaultStates()` method to return a set of [Slices configurations](https://redux-toolkit.js.org/api/createslice) (e.g. `initialState` & `reducers`).
+* The slices are identified by their key in the Slices definition object, and are prefixed with the component namespace to avoid conflicts (see example below).
+
+
+## Example:
+Defining and consuming a global state inside a component is fairly easy after you understand the basic concepts.
```

Comment:
No `you/we` here and below

Reply:
Every docs out there uses `you / we` when writing...

1. https://reactjs.org/docs/getting-started.html

> If **you’re** interested in playing around with React, **you** can use an online code playground. Try a Hello World template on CodePen, CodeSandbox, or Stackblitz.

> If **you** prefer to use **your** own text editor, **you** can also download this HTML file, edit it, and open it from the local filesystem in **your** browser. It does a slow runtime code transformation, so **we’d** only recommend using this for simple demos



2. https://laravel.com/docs/8.x

> Laravel is a web application framework with expressive, elegant syntax. A web framework provides a structure and starting point for creating **your** application, allowing **you** to focus on creating something amazing while **we** sweat the details.

> Whether **you** are new to PHP or web frameworks or have years of experience, Laravel is a framework that can grow with **you**. **We'll** help you take **your** first steps as a web developer or give **you** a boost as **you** take **your** expertise to the next level. **We** can't wait to see what **you** build.


Anyway, I chaned that...